ale_linters/tex/textidote.vim

```diff
@@ -0,0 +1,47 @@
+" Author: Jordi Altayo <jordiag@kth.se>, Juan Pablo Stumpf
+" <juanolon@gmail.com>
+" Description: support for textidote grammar and syntax checker
+
+call ale#Set('tex_textidote_executable', 'textidote')
+call ale#Set('tex_textidote_options', '--no-color --output singleline')
+" TODO get language from spell spelllang
+call ale#Set('tex_textidote_check_lang', '')
+
+
+function! ale_linters#tex#textidote#GetExecutable(buffer) abort
+    let l:exe = ale#Var(a:buffer, 'tex_textidote_executable')
+    let l:exe .= ' ' . ale#Var(a:buffer, 'tex_textidote_options')
+
+    let l:check_lang = ale#Var(a:buffer, 'tex_textidote_check_lang')
+
+    if !empty(l:check_lang)
+        let l:exe .= ' --check ' . l:check_lang
+    endif
+
+    return l:exe . ' ' . expand('#' . a:buffer . ':t')
+endfunction
+
+function! ale_linters#tex#textidote#Handle(buffer, lines) abort
+    let l:pattern = '.*' . expand('#' . a:buffer . ':t:r') . '\.tex(L\(\d\+\)C\(\d\+\)-L\d\+C\d\+): \(.*\)".*"'
```

Comment:
Same as previous comment this breaks for subfolder files. Remove the `:t` to make it work properly.